package.json

```diff
@@ -1337,12 +1322,12 @@
                 },
                 {
                     "command": "aws.cdk.renderStateMachineGraph",
-                    "when": "view == aws.cdk.explorer && viewItem == awsCdkStateMachineNode",
+                    "when": "viewItem == awsCdkStateMachineNode",
                     "group": "inline@1"
                 },
                 {
                     "command": "aws.cdk.renderStateMachineGraph",
-                    "when": "view == aws.cdk.explorer && viewItem == awsCdkStateMachineNode",
```

Comment:
No need to specify a view unless you really want such granular control. Or the context value is super generic for some reason and you think it might collide with other extensions.